Services/CO.CDP.Tenant.WebApi/Api/Tenant.cs

```diff
@@ -80,6 +81,7 @@ await useCase.Execute(urn)
                 operation.Responses["200"].Description = "Tenants associated with the the user.";
                 operation.Responses["401"].Description = "Valid authentication credentials are missing in the request.";
                 operation.Responses["404"].Description = "Tenant not found.";
+                operation.Responses["417"].Description = "Token sub is not found";
```

Comment:
This isn't the correct status code to return here. Don't we have 401 for this scenario?

[417](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/417) is for when the expectation given in the `Expect` header isn't met:

> The HTTP 417 Expectation Failed client error response code indicates that the expectation given in the request's [Expect](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect) header could not be met.